packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.html

```diff
@@ -7,255 +7,64 @@
-            data-header="{{:: ('ip_order_step1_question' | translate) + $ctrl.ADDITIONAL_IP}}"
-            data-valid="$ctrl.model.selectedService"
-            data-editable="!$ctrl.isLoading"
-            data-navigation="$ctrl.model.selectedService"
-            data-on-focus="$ctrl.onFlavorFocus()"
-            data-on-submit="$ctrl.manageLoadIpOffers()"
-        >
-            <div data-ng-show="$ctrl.loading.services" class="text-center">
-                <oui-spinner></oui-spinner>
-            </div>
-            <div
-                data-ng-show="!$ctrl.loading.services && $ctrl.services.length"
-            >
-                <oui-field
-                    data-label="{{:: 'ip_table_header_service' | translate }}"
-                    data-size="xl"
-                >
-                    <label class="oui-select">
-                        <select
-                            id="service"
-                            name="service"
-                            class="oui-select__input"
-                            data-ng-model="$ctrl.model.selectedService"
-                            data-ng-options="service as service.displayName group by $ctrl.getServiceTypeLabel(service.type) for service in $ctrl.services | orderBy:'displayName' track by service.serviceName"
-                        >
-                            <option
-                                data-ng-disabled="true"
-                                data-translate="global_select"
-                            >
-                            </option>
-                        </select>
-                        <span
-                            class="oui-icon oui-icon-chevron-down"
-                            aria-hidden="true"
-                        ></span>
-                    </label>
-                </oui-field>
-            </div>
-            <div
-                data-ng-show="!$ctrl.loading.services && !$ctrl.services.length"
-                data-translate="ip_order_noservices"
-            ></div>
-        </oui-step-form>
-        <oui-step-form
-            data-id="offer"
-            data-name="offer_selection"
-            data-header="{{:: 'ip_agora_table_header_offer' | translate }}"
-            data-navigation="$ctrl.model.params.selectedOffer"
-            data-on-cancel="$ctrl.resumeOrder()"
-            data-editable="!$ctrl.isLoading"
-            data-valid="$ctrl.model.params.selectedCountry"
-            data-submit-text="{{:: 'ip_agora_table_submit_text' | translate }}"
-            data-cancel-text="{{:: 'ip_agora_table_cancel_text' | translate}}"
-            data-prevent-next
+    </oui-header>
+
+    <div class="container-fluid d-flex flex-wrap">
+        <oui-select-picker
+            name="ip-version"
+            class="text-center d-flex col-sm-12 col-md-6 col-xl-3 my-3"
+            data-model="$ctrl.ipType"
+            data-variant="light"
+            data-on-change="$ctrl.$state.go('app.ip.agora-order.ipv4')"
```

Comment:
```suggestion
            data-on-change="$ctrl.goToIpv4Order()"
```
Do the redirection inside a controller function so we can add tracking in a second time